File: problemas/ad-hoc/1546/1546.rb
nomes = %w[Rolien Naej Elehcim Odranoel]

n = gets.to_i
for i in (1..n) do
  k = gets.to_i
  for j in (1..k) do
    feedback = gets.to_i
    puts nomes[feedback - 1]
  end
end

File: problemas/ad-hoc/1546/1546.exs
nomes = ["Rolien", "Naej", "Elehcim", "Odranoel"]

n = IO.gets("") |> String.trim() |> String.to_integer()
Enum.each(1..n, fn _ ->
  k = IO.gets("") |> String.trim() |> String.to_integer()
  Enum.each(1..k, fn _ ->
    feedback = IO.gets("") |> String.trim() |> String.to_integer()
    IO.puts Enum.at(nomes, feedback - 1)
  end)
end)